src/fragmentarium/ui/search/FragmentariumSearch.test.tsx

```diff
@@ -62,68 +59,93 @@ beforeEach(async () => {
   fragmentSearchService = new (FragmentSearchService as jest.Mock<
     jest.Mocked<FragmentSearchService>
   >)()
-  textService = new (TextService as jest.Mock<jest.Mocked<TextService>>)()
+  wordService = new (WordService as jest.Mock<jest.Mocked<WordService>>)()
   session = new MemorySession(['read:fragments'])
 })
 
+function parseMuseumNumber(museumNumber: string): MuseumNumber {
```

Comment:
Move this to MuseumNumber.ts

Reply:
In fact, we only ever use `museumNumber` in its string form, so I'll simplify the field to a string (equivalent to `Fragment.number`).